GVFS/GVFS/CommandLine/UpgradeVerb.cs

```diff
@@ -198,6 +198,18 @@ private bool TryRunProductUpgrade()
                     return false;
                 }
 
+                // If we are on a platform that does not support nuget verification, and user has not
+                // specified the --no-verify flag, then print a helpful message here.
+                if (!GVFSPlatform.Instance.UnderConstruction.SupportsNuGetVerification && !this.NoVerify)

```

Comment:
If the platform supports nuget verification should we disallow the `--no-verify` or are there use cases for that?

Reply:
The main use case is during development / authoring the upgrade package - it allows you test the upgrade package without running it through the full signing pipeline. I don't know of a use case for the mainline consumer.

I think it is useful enough for testing upgrade packages that it is worth leaving in (there are not that many switches) - but I don't feel strongly.

Reply:
I was mainly wondering about security and if leaving that option opens any attack vectors?

Reply:
@kewillford @jrbriggs this is the only remaining comment on this PR, and outside of it I think the changes in this PR are ready to go.

@kewillford, I don't think the scenario you've brought up is something new (introduced by these changes), correct?  I'm wondering if we can/should leave these changes as-is for now and follow a file up issue to determine if the `--no-verify` flag should be supported.

Thoughts?

Reply:
@wilbaker yep sounds good to me.

Reply:
Filed #1548 for this.